sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -1,13 +1,33 @@
 {
   "title": "Report Scholastic Standing Change",
-  "display": "form",
-  "type": "form",
   "name": "reportscholasticstandingchange",
   "path": "reportscholasticstandingchange",
+  "type": "form",
+  "display": "form",
   "tags": [
     "common"
   ],
   "components": [
+    {
+      "input": true,
+      "tableView": true,
+      "key": "applicationOfferingIntensity",
+      "label": "Offering Intensity (Part Time or Full Time) - Used to define custom labels",
+      "protected": false,
+      "unique": false,
+      "persistent": true,
+      "type": "hidden",
+      "tags": [],
+      "conditional": {
+        "show": "",
+        "when": null,
+        "eq": ""
+      },
+      "properties": {},
+      "lockKey": true,
+      "calculateValue": "if(value === \"Part Time\"){\n    // Find the option and adjust it to Part-time.\n    const scholasticStandingChangeType = utils.getComponent(form.components, \"scholasticStandingChangeType\");\n    const studentDidNotCompleteProgramOption = scholasticStandingChangeType.values.find(option => option.value === \"Student did not complete program\");\n    studentDidNotCompleteProgramOption.label = \"Student did not complete courses they were funded for\";\n}",
```

Comment:
Thanks for providing the formatted code. 

IMO using logic for this purpose will be more a suitable approach. 

![image](https://github.com/bcgov/SIMS/assets/54600590/e4f98090-c3fb-4669-b585-8c0cab054dcf)

Reply:
About the logic in the update, I can check if it will look better.
About the alternative of using another radio, do you mean another radio option or another full radio select component?
Consider that the two radio buttons will have the same "value" and the idea is to change the label.

Reply:
About using another radio I mentioned another full radio select, never mind. I tested and saw that even under condition it doesn't allow 2 components with same api key.

Reply:
About moving the logic into the update, I did the change locally as below and it looks better as you said.

![image](https://github.com/bcgov/SIMS/assets/61259237/44a6c8ba-1da2-41c0-9df7-8ee2a6fca2ab)

The code would be also reduced as below considering that now it is under the radio and the values can be accessed using `component.values` directly.
```js
const studentDidNotCompleteProgramOption = component.values.find(option => option.value === "Student did not complete program");
studentDidNotCompleteProgramOption.label = "Student did not complete courses they were funded for";
```